### PR TITLE
chore: bump Windows base images to 2024.9B

### PR DESCRIPTION
**Pull request by AbelHu**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
bump Windows base images to 2024.9B

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:

- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:

```
none
```

### CODE REVIEW

**Review by ShiqianTao — APPROVED** [review]

**Review by ShiqianTao — APPROVED** [review]

File: vhdbuilder/packer/generate-windows-vhd-configuration.ps1 [hunk 1]
```diff
@@ -72,9 +72,14 @@ switch -Regex ($windowsSku) {
         )
     }
     "23H2*" {
-        # Don't need to update patchUrls and patchIDs for 23H2 since it's by design.
-        $global:patchUrls = @()
-        $global:patchIDs = @()
+        if ($windowsSku -eq "23H2-gen2") {
+            # Don't need to update patchUrls and patchIDs for 23H2 since it's by design.
```
Comment (ShiqianTao) [inline vhdbuilder/packer/generate-windows-vhd-configuration.ps1:76]: this comment should be for lines 80-81 ?

File: vhdbuilder/packer/windows-image.env [hunk 2]
```diff
@@ -4,7 +4,7 @@
 # CLI example to get the latest image version:
 #   az vm image show --urn MicrosoftWindowsServer:WindowsServer:2019-Datacenter-Core-smalldisk:latest
 WINDOWS_2019_BASE_IMAGE_SKU=2019-Datacenter-Core-smalldisk
-WINDOWS_2019_BASE_IMAGE_VERSION=17763.6189.240811
+WINDOWS_2019_BASE_IMAGE_VERSION=17763.6293.240905
```
Comment (ShiqianTao) [inline vhdbuilder/packer/windows-image.env:7]: line 12 is 8B, should be updated to 9B

File: vhdbuilder/packer/generate-windows-vhd-configuration.ps1 (hunk 1 above)
Comment (AbelHu) [reply vhdbuilder/packer/generate-windows-vhd-configuration.ps1:76]: The command is for windows security patch feature. I think that we can remove it now.

File: vhdbuilder/packer/windows-image.env (hunk 2 above)
Comment (AbelHu) [reply vhdbuilder/packer/windows-image.env:7]: Good catch. Caused by rebasing.

**Review by ShiqianTao — APPROVED** [review]

**Review by junjiezhang1997 — APPROVED** [review]

**Pull request merged**